src/dataplane-protocol/tests/file_fetch.rs

```diff
@@ -115,8 +120,10 @@ async fn setup_client(addr: &str) -> Result<(), FlvSocketError> {
     assert_eq!(batches.len(), 1);
     let records = &batches[0].records();
     assert_eq!(records.len(), 2);
-    assert_eq!(records[0].value.to_string(), "record 0");
-    assert_eq!(records[1].value.to_string(), "record 1");
+    assert_eq!(records[0].value.to_string(), "value 0");
+    assert_eq!(records[0].key.as_ref().unwrap().to_string(), "key 0");
+    assert_eq!(records[1].value.to_string(), "value 1");
+    assert_eq!(records[1].key.as_ref().unwrap().to_string(), "key 1");
```

Comment:
records[1].key().unwrap().to_string()

Reply:
There is no `key()` method for `dataplane::record::Record`. Do you want me to add one?

Reply:
Yes.  we should add them